### PR TITLE
Adding tier-0 smoke test suite in 6.0 yaml file

### PR DESCRIPTION
**Pull request by rkhambad**
Signed-off-by: Rajendra Khambadkar <rkhambad@rkhambad.remote.csb>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by sunilangadi2 — DISMISSED** [review]
LGTM, But can you please add test result logs?

**Comment by rkhambad** [comment]
@sunilangadi2 
Test Log: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-HVRYTD/

Log is created after removing RDB I/O test cases as we have issue with rbd scripts

File: pipeline/metadata/6.0.yaml [hunk 1]
```diff
@@ -7,6 +7,22 @@
 # parameter required to create a similar custom file is suite name, suite yaml file, global configuration file,
 # platform, rhbuild, inventory and metadata information like frequency of execution, tier, cloud type, functional group and stage.
 #=====================================================================================
+- name: "Build smoke test for 6.0"
```
Comment (psathyan) [inline pipeline/metadata/6.0.yaml:10]: nit: Could we rename the test suite name `Smoke Test of RH Ceph build`

Comment (rkhambad) [reply pipeline/metadata/6.0.yaml]: Done.

**Comment by MohitBis** [comment]
LGTM

**Review by sunilangadi2 — APPROVED** [review]

**Review by psathyan — APPROVED** [review]

**Pull request merged**